src/components/common/ItemsNotFoundComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Image } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../../_css/ItemsNotFoundComponent.css';
+
+
+const ItemsNotFoundComponent = ({ message, header }) => (
+  <div className="not-found">
+    <div className="middle">
```

Comment:
It seems you didn't read anything about **BEM** methodology. Rename this class from `middle` to `not-found__header`, and don't forget to do the same in the `.scss` file